dear_petition/petition/models.py

```diff
@@ -210,6 +210,8 @@ class Offense(models.Model):
     )
     disposed_on = models.DateField(blank=True, null=True)
     disposition_method = models.CharField(max_length=256)
+    verdict = models.CharField(max_length=256, blank=True)
+    plea = models.CharField(max_length=256, blank=True)
```

Comment:
NOTE as a reminder for me while I review: Are we currently extracting `verdict` and `plea` from the JSONBlob yet? Can do we write to these fields?